blockly-ws-search/src/WorkspaceSearch.js

```diff
@@ -68,6 +68,13 @@ export class WorkspaceSearch {
      */
     this.searchText_ = '';
 
+    /**
+     * An optional custom search function to override default behavior.
+     * @type {?function(Blockly.WorkspaceSvg, string): Array.<Blockly.BlockSvg>}
+     * @protected
+     */
+    this.customSearchFunction_ = null;
```

Comment:
We might want to make this static. Blockly.WorkspaceSearch.customSearchFunction. Doesn't seem like there would be a large use case for changing on object by object basis.